tests/tainting_rules/java/tainted_obj.java

```diff
@@ -0,0 +1,9 @@
+class Test {
+    void test() {
+        Foo x = new Foo();
+        x.t = taint_source();
+        Foo y = x;
+        //ERROR:
+        sink(y.t, z);
```

Comment:
I'll take care of these cases as part of Python's index-sensitivity and restructuring, so you could pause this for now. I don't think we'll merge this PR.

Reply:
But don't delete it either, just in case!